bench/utils/__init__.py

```diff
@@ -0,0 +1,85 @@
+import struct
+import sys
+import time
+import random
+
+_RANDOM = random.Random(0)
```

Comment:
I think I have a better understanding of what you meant here, I will make the random generator be per-benchmark now rather than global. If this is not what you meant correct me for this is a small change.